v3/src/components/case-card/case-view.tsx

```diff
@@ -60,41 +126,107 @@ export const CaseView = observer(function InnerCaseView(props: ICaseViewProps) {
     })
   }
 
-  const renderChildCollection = (coll: ICollectionModel) => {
-    const childCases = cardModel?.groupChildCases(displayedCaseId) ?? []
-
-    return (
-      <ParentCollectionContext.Provider key={`${displayedCaseId}-${level}`} value={coll.parent?.id}>
-        <CollectionContext.Provider value={coll.id}>
-          <CaseView
-            cases={childCases}
-            level={level + 1}
-            onSelectCases={onSelectCases}
-            displayedCaseLineage={displayedCaseLineage}
-            onNewCollectionDrop={handleNewCollectionDrop}
-          />
-        </CollectionContext.Provider>
-      </ParentCollectionContext.Provider>
-    )
+  const classes = clsx(
+    "case-card-view",
+    colorCycleClass(level, collectionCount),
+    { "animating": animationStarted && !dummy }
+  )
+  const tileWidth = tileLayout?.width ?? 0
+  const leftLeft = `-${tileWidth}px`
+  const rightLeft = `${tileWidth}px`
+  const left = !isAnimating || animationStarted ? 0 : isFlippingRight ? leftLeft : rightLeft
+  const style = { left }
+  const otherLeft = !animationStarted ? 0 : isFlippingRight ? rightLeft : leftLeft
+  const otherStyle: React.CSSProperties = {
+    left: otherLeft,
```

Comment:
It seems like this applies to the dummy instance, in which case `dummyLeft` and `dummyStyle` would be clearer.